docker-stack-prod.yml

```diff
@@ -45,7 +45,7 @@ services:
             - back_network
         volumes:
             - /var/run/docker.sock:/var/run/docker.sock
-            - /volumes/whitebox_program_uploads:/uploads:ro
+            - /volumes/whitebox_program_uploads:/uploads
```

Comment:
A new folder (compilation folders for validating challenges) cannot be created if the volume is mounted in a read-only way.